var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -70,15 +87,17 @@ class Qt(Package):
     # https://github.com/spack/spack/issues/1517
     patch('qt5-pcre.patch', when='@5:')
 
-    patch('qt4-corewlan-new-osx.patch', when='@4')
-    patch('qt4-pcre-include-conflict.patch', when='@4')
-    patch('qt4-el-capitan.patch', when='@4')
-
-    # Allow Qt's configure script to build the webkit option with more
-    # recent versions of gcc.
-    # https://github.com/spack/spack/issues/9205
-    # https://github.com/spack/spack/issues/9209
-    patch('qt4-gcc-and-webkit.patch', when='@4')
+    patch('qt4-pcre-include-conflict.patch', when='@4.8.6')
+    patch('qt4-tools.patch', when='@4+tools')
+    if not MACOS_VERSION:
+        # Allow Qt's configure script to build the webkit option with more
+        # recent versions of gcc.
+        # https://github.com/spack/spack/issues/9205
+        # https://github.com/spack/spack/issues/9209
+        patch('qt4-gcc-and-webkit.patch', when='@4.8.6')
```

Comment:
Should this apply for `4.8.5` as well? You don't need to test if you don't know, but if you don't know then IMO it's worth applying this to both (since that's what was happening before).

Reply:
I don't know, so I'll make that change.